api/src/services/application.service.ts

```diff
@@ -350,6 +350,12 @@ export class ApplicationService {
         );
       }
     }
+    // if closed listing and non-admin user
```

Comment:
@YazeedLoonat Should this be in `authorizeAction`?

Reply:
zam, thats a good question

as it stands right now `authorizeAction` is more about leveraging the perms we have set up in casbin, and this is not really tied to casbin's perms at all

I think its okay to leave this in place, but as we look for a better permissioning system we should defo revisit